calico-vpp-agent/watchers/uplink_route_watcher.go

```diff
@@ -34,6 +43,22 @@ type RouteWatcher struct {
 	stop              bool
```

Comment:
`RouteWatcher.stop` should probably be changed to leverage the `tomb` passed to `WatchRoutes()`